### PR TITLE
Added grid to form

### PR DESCRIPTION
**Pull request by 3rfan**

### CODE REVIEW

**Comment by 3rfan** [comment]
@danrot

**Comment by danrot** [comment]
There is also another bug: If we have sections in sections, the inner section has no margin at all aftewards (The test section is the inner one in this screenshot):

image

I don't know if we want to support sections in sections, but if we do they should probably look different anyway. So I guess that could be ignored for now (?).

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Renderer.js [hunk 1]
```diff
@@ -33,27 +35,54 @@ export default class Renderer extends React.PureComponent<Props> {
         this.submitButton = submitButton;
     };
 
+    renderItem(schemaField: SchemaEntry, schemaKey: string) {
+        const {data, locale, onChange} = this.props;
+
+        if (schemaField.type === 'section') {
+            const {items} = schemaField;
+            return (
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Renderer.js]: extract into own method `renderGridSection`

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Renderer.js [hunk 2]
```diff
@@ -33,27 +35,54 @@ export default class Renderer extends React.PureComponent<Props> {
...
+                    </Grid.Item>
+                    {items &&
+                        Object.keys(items).map((key) => this.renderItem(items[key], key))
+                    }
+                </Grid.Section>
+            );
+        }
+
+        return (
+            <Grid.Item
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Renderer.js]: extract into own method `renderGridItem`

**Comment by chirimoya** [comment]
else ACK

File: src/Sulu/Bundle/AdminBundle/Resources/js/stores/ResourceMetadataStore/ResourceMetadataStore.js [hunk 3]
```diff
@@ -45,13 +45,43 @@ class ResourceMetadataStore {
...
+                address: {
+                    label: 'Address',
+                    type: 'section',
+                    items: {
+                        street: {
+                            label: 'Street',
+                            type: 'text_line',
+                            size: 6,
+                        },
+                    },
```
[the remark this answers is not in the document]
Comment (danrot) [reply src/Sulu/Bundle/AdminBundle/Resources/js/stores/ResourceMetadataStore/ResourceMetadataStore.js]: @chirimoya Should I remove the wrong testing parts?

[the remark this answers is not in the document]
Comment (chirimoya) [reply src/Sulu/Bundle/AdminBundle/Resources/js/stores/ResourceMetadataStore/ResourceMetadataStore.js]: yes

**Pull request merged**